content/en/agent/basic_agent_usage/windows.md

```diff
@@ -23,26 +23,38 @@ This page outlines the basic features of the Windows Datadog Agent. If you haven
+| `LOGS_ENABLED`             | String | Enable (`"true"`) or disable (`"false"`) the log collection feature in the configuration file. Logs are disabled by default.                                                                                                      |
+| `APM_ENABLED`              | String | Enable (`"true"`) or disable (`"false"`) the APM Agent in the configuration file. APM is enabled by default.                                                                                                                      |
+| `PROCESS_ENABLED`          | String | Enable (`"true"`) or disable (`"false"`) the Process Agent in the configuration file. The Process Agent is disabled by default.                                                                                                   |
+| `CMD_PORT`                 | Number | A valid port number between 0 and 65534. The Datadog Agent exposes a command API on port 5001. If that port is already in use by another program, the default may be overridden here.                                             |
+| `PROXY_HOST`               | String | If using a proxy, sets your proxy host. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                               |
+| `PROXY_PORT`               | Number | If using a proxy, sets your proxy port. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                               |
+| `PROXY_USER`               | String | If using a proxy, sets your proxy user. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                               |
+| `PROXY_PASSWORD`           | String | If using a proxy, sets your proxy password. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                           |
+| `DDAGENTUSER_NAME`         | String | Override the default `ddagentuser` username used during Agent installation _(v6.11.0+)_. [Learn more about the Datadog Windows Agent User][3]                                                                                     |
+| `DDAGENTUSER_PASSWORD`     | String | Override the cryptographically secure password generated for the `ddagentuser` user during Agent installation _(v6.11.0+)_. Must be provided for installs on Domain Servers. [Learn more about the Datadog Windows Agent User][3] |
+| `APPLICATIONDATADIRECTORY` | Path   | Override the directory to use for the configuration file directory tree. May only be provided on initial install; not valid for upgrades. Default: `c:\programdata\datadog`. _(v6.11.0+)_                                         |
```

Comment:
Default is the system default which might change following the box setting. User might have set another disk or folder as  the system wide `programdata`.  The default is to use `%PROGRAMDATA%\datadog`
WDYT @derekwbrown ?

Same for `PROJECTLOCATION` with `%PROGRAMFILES%\Datadog\Datadog Agent`

Reply:
good point, I'll let @derekwbrown chime in here.

Note that `c:\programdata` and `c:\Program Files\Datadog\Datadog Agent` are mentioned in many other locations in the documentation, if we want to update it here we should likely update it elsewhere as well, and decide whether it'll be clear enough for must users or if we should also document that `%PROGRAMFILES%` typically defaults to `c:\Program Files\Datadog\Datadog Agent` for example

Reply:
(in any case, should we address this later?)